documents/forOpenAPISpecification/sample_divided/pets/pets.yaml

```
@@ -0,0 +1,167 @@
+get:
+  summary: FunctionID2
+  description: List all pets
+  operationId: get-pets

```

operationIdはcamelCaseかと。
https://future-architect.github.io/coding-standards/documents/forOpenAPISpecification/OpenAPI_Specification_3.0.3.html#paths-path-method-operationid

```suggestion
  operationId: searchPets
```

---

ありがとうございます。迷いましたが、ここでは規約に記載の「HTTP メソッドと URL パスの組み合わせをキャメルケースで表現」とさせて頂きました。もし変更したほうがよさそうなら共有いただきたくよろしくお願いいたします。